devel/example_workflow.R

```diff
@@ -15,27 +15,40 @@ git_stats <- create_gitstats() %>%
   )
 
 # examples for getting repos (default argument for parameter 'by' is 'org')
```

Comment:
I would add `pkgload::load_all()` at the beginning for faster runs, but that's my personal preference